input/fsh/who-cr-CompositionCaseReportBasic.fsh

```diff
@@ -0,0 +1,261 @@
+Title: "WHO CR Encounter"
+Description: "WHO Encounter for a case report"
+* location 1..1
+* reasonCode 1..*
+* reasonCode from WhoCrValueSetReasonForCovid19Testing
+
+Profile: WhoCrLocation
+Parent: Location
+Id: who-cr-location
+Title: "WHO CR Location"
+Description: "WHO Location for case report"
+* address 1..1
+* address.country 1..1
+* address.state 1..1
+
+Profile: WhoCrObservationAge
+Parent: Observation
+Id: who-cr-observation-age
+Title: "WHO CR Observation Age"
+Description: "WHO Observation of age for case report"
+* code = $LNC#30525-0
+* value[x] only Quantity
+* valueQuantity.value 1..1
+* valueQuantity from WhoCrValueSetAgeUnits
+
+Profile: WhoCrObservationBirthSex
+Parent: Observation
+Id: who-cr-observation-birth-sex
+Title: "WHO CR Observation Birth Sex"
+Description: "WHO Observation of birth sex for case report"
+* code = $LNC#76689-9
+* value[x] only CodeableConcept
+* valueCodeableConcept 1..1
+* valueCodeableConcept from WhoCrValueSetSexAtBirth
+
+Profile: WhoCrObservationLabTestDate
+Parent: Observation
+Id: who-cr-observation-lab-test-date
+Title: "WHO CR Observation Lab Test Date"
+Description: "WHO Observation for lab test date"
+* code = http://test.org/obsCode#lab-test-date // TODO: code not defined in questionnaire either
+* value[x] only dateTime
+* valueDateTime 1..1
+
+Profile: WhoCrObservationSymptoms
+Parent: Observation
+Id: who-cr-observation-symptoms
+Title: "WHO CR Observation Symptoms"
+Description: "WHO Observation if the patient has any symptoms for case report"
+* code = http://test.org/obsCode#symptoms // TODO: code not defined in questionnaire either
+* value[x] only CodeableConcept
+* valueCodeableConcept 1..1
+* valueCodeableConcept from WhoCrValueSetYesNo
+
+Profile: WhoCrObservationOnsetDate
+Parent: Observation
+Id: who-cr-observation-onset-date
+Title: "WHO CR Observation Symptoms Onset Date"
+Description: "WHO Observation for date of symptoms onset"
+* code = $LNC#65222-2
+* value[x] only dateTime
+* valueDateTime 1..1
+
+Profile: WhoCrObservationComorbidities
+Parent: Observation
+Id: who-cr-observation-comorbidities
+Title: "WHO CR Observation Co-morbidities"
+Description: "WHO Observation if the patient has any co-morbidities for case report"
+* code = $LNC#75618-9
+* value[x] only CodeableConcept
+* valueCodeableConcept 1..1
+* valueCodeableConcept from WhoCrValueSetYesNo
+
+Profile: WhoCrObservationHasTravelled
+Parent: Observation
+Id: who-cr-observation-has-travelled
+Title: "WHO CR Observation Has Travelled"
+Description: "WHO Observation if the patient has travelled in the past 14 days prior to symptom onset for case report"
+* code = $LNC#96542-6
+* value[x] only CodeableConcept
+* valueCodeableConcept 1..1
+* valueCodeableConcept from WhoCrValueSetYesNo
+
+/**
+ * Examples
+ */
+
+Instance: WhoCrBundleExample
+InstanceOf: Bundle
+Usage: #example
+Title: "Who Cr Bundle Example"
+Description: "Example of a clinical bundle representing a case report"
+* type = #document
```

Comment:
@de-laz the should be a #transaction - the document Bundle won't create the individual resources as we need them. If we need to get all the related documents together again in a clinical context this would be done with the composition resource which has a reference to all related resources